policy/report.go

```diff
@@ -66,9 +66,16 @@ func (s *Stats) Add(score *Score) {
 }
 
 func (sd *ScoreDistribution) Add(score *Score) {
-	sd.Total++
+	sd.AddRating(score.Rating())
```

Comment:
do we need to catch the case where score is nil?

Reply:
That makes sense, however looking at the `Rating()` function on the score it already handles a nil score and returns `ScoreRating_Unrated`. 

Seeing as how that function is used in a few places I'd prefer to not introduce that check right now since I do not know how scores are being passed in wherever that function is being used, it could lead to a change in behaviour.

Reply:
Makes sense, please add a comment to the function. Essentially it allows nil scores and counts them accordingly. With is all good. So if the caller cares about that it need to handle nil values themselves.

Reply:
also updated the tests so that if we change this in the future, there will be a failing test to indicate that